ateam_common/src/game_controller_listener.cpp

```diff
@@ -106,91 +108,81 @@ void GameControllerListener::RefereeMessageCallback(
 
   game_stage_ = static_cast<GameStage>(msg->stage);
 
-  uint8_t command = msg->command;
+  game_command_ = ConvertGameCommand(msg->command);
+  if (game_command_ != cached_game_command_) {
+    prev_game_command_ = cached_game_command_;
+    cached_game_command_ = game_command_;
+  }
```

Comment:
What's the reason we cache the game command?

Reply:
Hmmm we don't actually need that to be a distinct variable I think. I can massage this code a bit to get rid of that while keeping the same behavior.